cmd/package_managers.go

```diff
@@ -108,6 +141,36 @@ func fetchGitRepositoryFromNPM(packageName string, packageManager pmc.Client) (s
 	return v.Objects[0].Package.Links.Repository, nil
 }
 
+func findGitRepositoryInPYPIResponse(packageName string, response io.Reader) (string, error) {
+	v := &pypiSearchResults{}
+	err := json.NewDecoder(response).Decode(v)
+	if err != nil {
+		return "", sce.WithMessage(sce.ErrScorecardInternal, fmt.Sprintf("failed to parse pypi package json: %v", err))
+	}
+
+	v.Info.ProjectURLs["key_not_used"] = v.Info.ProjectURL
```

Comment:
Technically this could collide with a key from the repos `project_urls` field (though unlikely)? Not sure if the empty string is a better alternative? Not sure there's any one best answer here.

Reply:
Yea the empty quotes felt worse, so I made the key longer =\

Happy to do whatever

Reply:
Let's leave it as is. Collision is unlikely, and worst case someone files an issue one day and we fix it down the line.